todoman/ui.py

```diff
@@ -295,6 +324,19 @@ def format_datetime(self, dt):
             self._format_time(time_part)
         )))
 
+    def parse_priority(self, priority):
+        if priority not in ['low', 'high', 'medium', 'none']:
```

Comment:
Actually, if it's not on this list, we should fail. We might also want to accept numbers as input (I don't have a strong opinion on this though).